src/components/GenreSelect.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+class GenreSelect extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
I propose to operate with the selected genre without creating a state inside the component. Use props.